pyproject.toml

```diff
@@ -40,17 +39,21 @@ version_file = "src/jax_finufft/jax_finufft_version.py"
 [tool.cibuildwheel]
 skip = "pp* *-musllinux_* *-manylinux_i686"
 build-verbosity = 1
-config-settings = {"cmake.define.FINUFFT_ARCH_FLAGS" = ""}
+config-settings = { "cmake.define.FINUFFT_ARCH_FLAGS" = "" }
 
 [tool.cibuildwheel.linux]
 before-all = "yum install -y fftw-devel"
 
 [tool.cibuildwheel.macos]
-before-all = "brew install fftw"
+before-all = "brew install llvm libomp"
```

Comment:
If we aren't going to try to target older `MACOSX_DEPLOYMENT_TARGET`, then I think we can just brew-install fftw. Trying to support old targets was the reason I removed this. (Likewise, all the `CC=...` environment variable setup below may not be necessary anymore, but it probably doesn't hurt either.)